policyengine_us/tests/policy/baseline/gov/states/ri/tax/income/exemption/ri_exemption.yaml

```diff
@@ -0,0 +1,53 @@
+- name: Filer claims full amount of exemption
+  period: 2022
+  input:
+    state_code: RI
+    adjusted_gross_income: 200_000
+    tax_unit_size: 1
+  output:
+    ri_exemption: 4_350
+
+- name: Filer claims limited amount of exemption
```

Comment:
each test has to have a unique name that explains the situation f.e. household with 2 exemptions and income of 217,100